test-lib.sh

```diff
@@ -1186,4 +1186,58 @@ function ct_timestamp_diff() {
   date -u -d "0 $final_date seconds - $start_date seconds" +"%H:%M:%S"
 }
 
+# ct_get_certificate_timestamp
+# ----------------------------
+# Looks into a running container into a specified file (certificate) and extracts
+# a notBefore date.
+# Argument: container - ID of a running container
+# Argument: path - path inside the running container
```

Comment:
Is this path to **a certificate** inside the dockerfile?

Reply:
yes, will add this to the comment